packages/api/tests/models/test-pdrs-model.js

```diff
@@ -292,6 +332,46 @@ test(
   }
 );
 
+test(
+  'storePdrFromCumulusMessage does not update if PDR record is from an older, prior completed execution',
+  async (t) => {
+    const pdrName = randomId('pdr');
+    const stateMachine = randomId('parsePdr');
+    const execution = randomId('exec');
+
+    const initialMsg = createPdrMessage({
+      execution,
+      stateMachine,
+      numCompletedExecutions: 3,
+      status: 'completed',
+      createdAtTime: (Date.now() + 10000000000),
+    });
+
+    initialMsg.payload.pdr = {
+      name: pdrName,
+    };
+
+    await pdrsModel.storePdrFromCumulusMessage(initialMsg);
+
+    const newMsg = createPdrMessage({
+      execution: randomId('newExec'),
+      stateMachine,
+      numRunningExecutions: 2,
+      status: 'failed',
```

Comment:
Same here, I think we should pass in `createdAtTime` to be sure that this message test fixture is older than the one previously created in this test

Reply:
8ecae9657